[WIP] docs(common): Prefix and Suffix documentation (#1828)
telerik/blazor-docs: _contentTemplates/common/inputs.md

Consider setting `DebounceDelay="0"` to the component inside the editor template. This is how the default editors in all Telerik Blazor components work. Otherwise, fast users may try to save changes before the data item in edit mode receives the new value.

#end

#adornments
## Adornments

> The component allows adding custom elements as prefix and suffix. [Read more about how to render custom adornments before and after the input element...]({%slug common-features/input-adornments%})

Reviewer: ```suggestion
The component allows you to add custom elements as prefixes and suffixes. [Read more about how to render custom adornments before and after the input element...]({%slug common-features/input-adornments%})
```

#end

#floating-label-and-preffix
When using the [`PrefixTemplate`]({%slug common-features/input-adornments%}#adding-prefix-in-ui-for-blazor-components) for a component wrapped in a [FloatingLabel]({%slug floatinglabel-overview%}), the label will overlap the prefix.

> To ensure both the FloatingLabel and the prefix content are properly displayed, configure the label position like so:

Reviewer 2: "configure the label position like so" does not bring any value. If there is one thing to do, you can mention it directly:

```suggestion
To ensure both the FloatingLabel and the prefix content are properly displayed, move the label with CSS:
```
